Instruments/microwave_tuning_gui.py

```diff
@@ -28,8 +28,9 @@
 import numpy as np
 
 class TuningWindow(QMainWindow):
-    def __init__(self, B12, parent=None):
+    def __init__(self, B12, config, parent=None):
```

Comment:
by pulling in the config dict we can write to and pull parameters from the ini file!

Reply:
we don't want that